src/Controller/Admin/ConferenceController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Controller\Admin;
+
+use AlterPHP\EasyAdminExtensionBundle\Controller\EasyAdminController;
+use App\Entity\Conference;
+use Symfony\Component\Form\Form;
+
+class ConferenceController extends EasyAdminController
+{
+    protected function persistNextConferenceEntity(Conference $conference, Form $newForm): void
+    {
+        if ($conference->isOnline() && null === $conference->getCity()) {
+            $conference->setCity('Online');
+        }
+
+        parent::persistEntity($conference);
+    }
```

Comment:
Actually, I would made the change here: https://github.com/jolicode/starfleet/blob/master/src/Entity/Conference.php#L379

    public function setOnline(bool $online): self
    {
        $this->online = $online;
        $this->city = 'Online';
  
        return $this;
    }